ppr-ui/src/composables/mhrInformation/useTransportPermits.ts

```diff
@@ -263,7 +275,8 @@ export const useTransportPermits = () => {
         reserveNumber: '',
         exceptionPlan: ''
       } as MhrRegistrationHomeLocationIF,
-      ownLand: null
+      ownLand: null,
+      registrationStatus: ''
```

Comment:
@cameron-eyds just a note here, this is original status for the registration, might not be the best place...